src/utils/helpers/forgot-password-validation.js

```diff
@@ -0,0 +1,52 @@
+import * as yup from 'yup'
+
+const valueIsNotEmpty = (value) => value?.trim()
```

Comment:
Instead of creating new function you just can export and import already  created function.
Try to do it please.